Tests/test_node_editor.py

```
@@ -84,9 +84,9 @@ def test_link_callback(self):
         node_id2 = self.node_editor.builder.build_node(anode2, "node_editor")
         node_id3 = self.node_editor.builder.build_node(anode3, "node_editor")
 
-        node1: AbstractNode = dpg.get_item_user_data(node_id1)
-        node2: AbstractNode = dpg.get_item_user_data(node_id2)
-        node3: AbstractNode = dpg.get_item_user_data(node_id3)
+        node1: AbstractNode = get_userdata(node_id1)
```

Ruff (0.15.20)

[error] Undefined name `get_userdata`

(F821)